source/firmware/xilinx-tools/flashing.md

```diff
@@ -29,10 +33,20 @@ Xilinx provides a First-Stage Bootloader application project which we will use t
 
 4. Ensure popup menu settings look like the following.
 
-Explaination of settings: `.MCS` is the file format which is supported for QSPI flashing. The list of three items for "Boot image partitions" must always be the following in this order: fsbl.elf, FPGA bitstream .bit file, your user *.elf file.
+Explaination of settings: `.MCS` is the file format which is supported for QSPI flashing. The list of three items for "Boot image partitions" must always be the following in this order: fsbl.elf, FPGA bitstream .bit file, your user *.elf file. 
+
+Skip below for dual core project procedure.
 
 ![Popup menu settings](images/flashing/img2.png)
 
+For dual core projects, there will be multiple *.elf files*. The order is important! 
+* Navigate to your `$REPO_ROOT\AMDC-Firmware\sdk\app_cpu0\Debug` folder and locate the file `app_cpu0.elf`. 
+    * Add this to the _Boot Image Partitions_ area
+* Navigate to your `app_cpu1` location, usually this is located in the `$REPO_ROOT\AMDC-Private\Debug` folder.
+    * Add the *.elf* file to your _Boot Image Partitions_ after the `app_cpu0.elf` file. 
+
+![Popup menu settings](images/flashing/img2-2.PNG)
```

Comment:
Can you put this in an [admonition](https://pradyunsg.me/furo/kitchen-sink/admonitions/) call-out below the img2.png file? The user should be drawn to this dual core stuff.

For examples of admonitions, see other doc `*.md` files which use them

Reply:
Done